doc/language/spec.html

```diff
@@ -123,13 +123,17 @@ <h2 id="lexing">Lexing</h2>
 <li><i>number</i>: As defined by <a href="http://json.org/">JSON</a> but without the leading
 minus.</li>
 
-<li><i>string</i>: Which can have three quoting forms:
+<li><i>string</i>: Which can have five quoting forms:
 
 <ul>
 <li>Double-quoted, beginning with <code>"</code> and ending with the first subsequent non-quoted
 <code>"</code> </li>
 <li>Single-quoted, beginning with <code>'</code> and ending with the first subsequent non-quoted
 <code>'</code> </li>
+<li>Double-quoted verbatim, beginning with <code>@"</code> and ending with the first subsequent
```

Comment:
You should probably say "non-quoted" again here, although the quoting scheme is different, it's still a quoting scheme.